web/client/components/data/query/QueryToolbar.jsx

```diff
@@ -103,6 +105,8 @@ const QueryToolbar = React.createClass({
             sortOptions: this.props.sortOptions,
             hits: this.props.hits
         };
+        this.props.actions.closeDrawer();
```

Comment:
closeDrawer and disableDrawer should be handled in onQuery. QueryToolbar does not know it's included in a drawer